test/unit-test/sigv4_config.h

```diff
@@ -76,7 +76,7 @@
  * <b>Default value:</b> `1024`
  */
 #ifndef SIGV4_PROCESSING_BUFFER_LENGTH
-    #define SIGV4_PROCESSING_BUFFER_LENGTH    350
+    #define SIGV4_PROCESSING_BUFFER_LENGTH    428
```

Comment:
This seems very arbitrary. Why 428?

Reply:
I changed the buffer for the new header `X_AMZ_CONTENT_SHA256` included in the canonical and signed headers.